### PR TITLE
Add image source: desktoppr.com

### PR DESCRIPTION
**Pull request by nkprince007**
Images are sourced using api.desktoppr.com

Resolves issue #212

### CODE REVIEW

File: uiplib/scrape.py [hunk 1]
```diff
@@ -90,6 +90,31 @@ def get_reddit_image_links(url, no_of_images):
...
+    returns a list of tuples, (filename, image_url)
+    """
+    responses = []
+    image_links = []
+    index = 1
+
+    while len(responses) < no_of_images:
+        page_url = url + ('?page=%d' % index)
+        page = requests.get(page_url)
+        data = json.loads(page.text)
```
Comment (abhsag24) [inline uiplib/scrape.py]: use `make_json`?

Comment (nkprince007) [inline uiplib/scrape.py]: But that method is defaulted to append `.json` to the url

**Comment by abhsag24** [comment]
Ahh then can you change it and corresponding functions to have a more
generic behaviour?

**Comment by nkprince007** [comment]
Sure, bro.

**Comment by abhsag24** [comment]
@nkprince007  looks really good, can you just add the entry for gui so that we are consistent?

**Comment by nkprince007** [comment]
Yeah, forgot about it... :P will get it fixed in a moment. :+1:

**Comment by abhsag24** [comment]
@nkprince007 Looks good feel free to merge. Also remember to open issue to use variable instead of constant values in GUI

**Comment by nkprince007** [comment]
Sure bro.. @abhsag24

**Pull request merged**